apps/hip/templates/includes/right_scroll_nav.html

```diff
@@ -0,0 +1,25 @@
+{% load wagtailcore_tags %}
+<!-- Right Scroll Navigation for Anchor Links on desktop only
```

Comment:
Rather than using HTML comment syntax, we could use [django template multi-line comment syntax](https://docs.djangoproject.com/en/3.1/ref/templates/builtins/#comment) so that this doesn't get rendered.